src/web.ts

```diff
@@ -0,0 +1,176 @@
+import { Probot, Server } from "probot";
+
+import { Config, loadConfig } from "./config";
+import { Database, PRAction } from "./db";
+import { createFailureIssue } from "./github";
+import { newLogger } from "./log";
+
+/**
+ * Add the handlers to Probot app.
+ * @param {Probot} app the Probot app to configure.
+ * @param {Config} config the Magic Mirror configuration.
+ * @param {Database} db the Magic Mirror database.
+ */
+export async function app(app: Probot, config: Config, db: Database) {
```

Comment:
nit: with the function called `app`, should the first arg also be called `app`? If that's a common pattern in Probot or something, that's fine, but to me it seems a little odd. Could the arg be something like `bot`?

Reply:
I used the naming conventions from Probot but I agree. I'll rename the argument to `probot`.